airsonic-main/src/main/java/org/airsonic/player/service/MediaScannerService.java

```diff
@@ -44,7 +44,6 @@
 @Service
 public class MediaScannerService {
 
-    private static final int INDEX_VERSION = 15;
```

Comment:
Now that IndexManager has been created, it is appropriate to handle INDEX_VERSION with IndexManager.
This variable has moved.